### ISSUE TITLE
Description containing some tables fails to migrate (ADF)

### ISSUE THREAD

**Issue by eplnde** [solidify/jira-azuredevops-migrator#961]
**Describe the problem**

If the Jira description field contains tables with "merged cells", the are lost/dropped

**To Reproduce**

1. Paste an excel table with combined cells in to the description field
2. Run jira export
3. check the Json output

![image](https://github.com/solidify/jira-azuredevops-migrator/assets/141833384/ce52cd44-9bed-4662-927e-d09d0e620e8d)

otput:
        {
          "ReferenceName": "System.Description",
          "Value": "<p>For test of some AzDo migration issues</p>\n\n"
        },

**Tool version**
 - Think I am rebased towards 3.0.331 at the moment

**Attachments**

Please attach the following files:

- config.json
- jira-export-log.txt
- wi-import-log.txt

**Screenshots**

**Comment by eplnde** [comment]
Ah, I see even the output when pasted here rendered weird, this is a screenshot:

![image](https://github.com/solidify/jira-azuredevops-migrator/assets/141833384/05aaedd1-b925-471c-b41f-b81eee4e544e)

**Comment by Alexander-Hjelm** [comment]
@eplnde I assume we are talking about the standard tables in jira? See screenshot:

![image](https://github.com/solidify/jira-azuredevops-migrator/assets/10683896/899244be-c106-4cf6-870b-56b7b00932b2)

It seems like this is some kind of macro inside jira, and the data is never exported as proper html. So maybe there is another endpoint involved to get the html formatted table?

Marking as feature request.

**Comment by eplnde** [comment]
No, this is a cust&paste table from excel
I do not think you can merge/combine cells in the Jira table-interface

**Comment by Alexander-Hjelm** [comment]
Ah ok!

Either way this feature will need to be implemented.

**Comment by eplnde** [comment]
@Alexander-Hjelm I think it would be good to at least add some warning that data is dropped as a "quick fix"
Right now this easily slips under the radar

**Comment by Alexander-Hjelm** [comment]
We did some repro and we found that regular tables in jira ([see my screenshot](https://github.com/solidify/jira-azuredevops-migrator/issues/961#issuecomment-1884830747)) are migrating just fine.

It must be that bringing in table data from external sources like excel causes a problem.

Will look into this asap, but this is not our highest priority at the moment.

@eplnde, maybe there is a way to convert your data, like first creating a regular table in Jira and then pasting the data? Even if that is not optimal if you have a lot of data, of course. I'm not sure myself here.

**Comment by eplnde** [comment]
You need to merge two cells in excel before pasting the table in to Jira to be able to reproduce the issue

i.e. this function:
![image](https://github.com/solidify/jira-azuredevops-migrator/assets/141833384/16856195-8edd-4f43-9068-a67fc61276db)

So not that common, but when you merge thousands of issues, it is easy to miss that a table was lost

**Comment by Alexander-Hjelm** [comment]
Aha ok!

This is an excel function, right? I am guessing that default HTML tables in Jira do not support this function?

**Comment by eplnde** [comment]
yes, as mentioned in https://github.com/solidify/jira-azuredevops-migrator/issues/961#issuecomment-1884845175
_"I do not think you can merge/combine cells in the Jira table-interface"_
